### DISCUSSION TITLE
What's prepending an @ to my $PS1?

### DISCUSSION THREAD

**Discussion by Strykar** [kovidgoyal/kitty#5729] [Q&A] [answered]
In `.bashrc` I have set `PS1='[\u@\h \W]\$ '` but my prompt looks like `@[strykar@i-5 ~]$` instead of `[strykar@i-5 ~]$`.
```
@[strykar@i-5 ~]$ echo $PS1
\[\e]133;k;start_kitty\a\]\[\e]133;A\a\]\[\e]133;k;end_kitty\a\][\u@\h \W]\$ \[\e]133;k;start_suffix_kitty\a\]\[\e[5 q\]\[\e]2;\w\a\]\[\e]133;k;end_suffix_kitty\a\]
@[strykar@i-5 ~]$ 
```
How do I figure out what's inserting this?

**Comment by kovidgoyal** [comment]
Works fine for me with kitty 0.26.5 and bash 5.1.16

**Comment by Strykar** [comment]
Running kitty 0.26.5 and Bash 5.1.016 myself :confused:
Where would I even begin to troubleshoot this?

**Comment by kovidgoyal** [comment]
bisect your bashrc

**Comment by Strykar** [comment]
Mine's not big or complex, unless [ble.sh](https://github.com/akinomyoga/ble.sh) is mucking with it - https://gist.github.com/Strykar/c91be98fc0128b997c030a2bd7a07f3b

**Comment by Strykar** [comment, marked as answer]
Issue was me having added `set show-mode-in-prompt on` in `.inputrc` years ago and forgetting about it.
Closing.
